tests/integration/mininet_tests.py

```diff
@@ -3819,7 +3819,7 @@ def test_untagged(self):
                 '10.0.0.0/24 next-hop 10.0.0.254',
                 '10.0.1.0/24 next-hop 10.0.0.1',
                 '10.0.2.0/24 next-hop 10.0.0.2',
-                '10.0.2.0/24 next-hop 10.0.0.2'):
+                '10.0.3.0/24 next-hop 10.0.0.2'):
```

Comment:
Should the next hop be 10.0.0.3 here?

Reply:
No I don't believe so, as the config used for this test-case has:
```
            - route:
                ip_dst: "10.0.3.0/24"
                ip_gw: "10.0.0.2"
```

Reply:
ah, good catch. I was just following the pattern and made an assumption that it should be 3.